model/project_configs.go

```diff
@@ -14,12 +14,17 @@ import (
 
 type ProjectConfig struct {
 	Id         string    `yaml:"_id" bson:"_id"`
-	Project    string    `yaml:"project,omitempty" bson:"project,omitempty"`
 	CreateTime time.Time `yaml:"create_time,omitempty" bson:"create_time,omitempty"`
+	// ProjectConfigFields are the properties on the project config that do not duplicate parser project's fields to allow strict unmarshalling of a full config file.
+	// Since a config file gets split into ParserProject and ProjectConfig, strict unmarshalling does not work when duplicate fields exist (e.g. Id, CreateTime).
+	ProjectConfigFields `yaml:",inline" bson:",inline"`
+}
 
+type ProjectConfigFields struct {
 	// These fields can be set for the ProjectRef struct on the project page, or in the project config yaml.
 	// Values for the below fields set on the project page will take precedence over this struct and will
 	// be the configs used for a given project during runtime.
+	Project                string                         `yaml:"project,omitempty" bson:"project,omitempty"`
```

Comment:
I think Project doesn't belong here since it's not a user configured field but instead an Evergreen populated field; however I don't think we need this at all -- do we use it for anything? Might be worth removing this field.

Reply:
I can take it out of the ProjectConfigFields struct but I think we still do need it - it actually acts as an "identifier" field for ProjectConfig.